src/dataExplorer/components/ResultsPane.tsx

```diff
@@ -120,9 +115,10 @@ const ResultsPane: FC = () => {
     selection,
     resource,
   } = useContext(PersistanceContext)
+  const orgID = useSelector(getOrg).id
```

Comment:
This is a potentially dangerous expression, since [`getOrg`](https://github.com/influxdata/ui/blob/master/src/organizations/selectors/index.ts#L22) can return `null`. This seems very unlikely, since if there's no org in state, it probably shouldn't ever be able to get to this point and there are probably larger issues going on. But I still wanted to call it out, just in case.

Reply:
Good call!